soc/arm64/intel_socfpga/agilex/mmu_regions.c

```diff
@@ -26,6 +26,12 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_SIZE(DT_NODELABEL(uart0)),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_DEFAULT_SECURE_STATE),
 
+	MMU_REGION_FLAT_ENTRY("PINMUX",
```

Comment:
I'm not super familiar with this subject, so perhaps @gmarull could take a look.